utilities/pipelines/dependencies/Microsoft.KeyVault/vaults/parameters/nopr.parameters.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "adp-<<namePrefix>>-az-kv-nopr-001"  // keyvault with purge protection enabled, to be used for encryption
+    },
+    "accessPolicies": {
+      "value": [
+        {
+          "tenantId": "<<tenantId>>",
+          "objectId": "bab643ad-2988-4560-9f2b-c8282ba345f0", // Backup Management Service needs access to back up the keys and secrets, along with the associated VMs
+          "permissions": {
+            "keys": [
+              "Get",
+              "List",
+              "Backup"
+            ],
+            "secrets": [
+              "Get",
+              "List",
+              "Backup"
+            ],
+            "certificates": []
+          }
+        },
+        {
+          "tenantId": "<<tenantId>>",
+          "objectId": "<<msiPrincipalID>>", // The object ID of the deployed MSI. Replaced by the pipeline
+          "permissions": {
+            "keys": [
+              "Get",
+              "WrapKey",
+              "UnwrapKey"
+            ],
+            "secrets": [
+              "Get"
+            ],
+            "certificates": []
+          }
+        },
+        {
+          "tenantId": "<<tenantId>>",
+          "objectId": "<<deploymentSpId>>", // dependency access
+          "permissions": {
+            "keys": [
+              "all"
+            ],
+            "secrets": [
+              "all"
+            ],
+            "certificates": [
+              "all"
+            ]
+          }
+        }
+      ]
+    },
+    "enablePurgeProtection": {
```

Comment:
Please add the softDeleteRetentionInDays property and set its value to the minimum

Reply:
added softDeleteRetentionInDays=7 on a new KV "adp-<<namePrefix>>-az-kv-nopr-002" as the property cannot be modified when it is already in place.
Deleted adp-<<namePrefix>>-az-kv-nopr-001.